1-quickstart/README.md

```diff
@@ -52,7 +55,93 @@ docker build -t mass-ingest:quickstart --build-arg MODERNE_CLI_VERSION=3.50.0 ..
 
 ### 3. Run the container
 
-Credentials are configured at runtime (not baked into the image):
+Credentials are configured at runtime (not baked into the image). You can use S3, Artifactory, or any Maven-compatible repository.
+
+#### Option A: Using S3 Storage
+
+For S3 or S3-compatible storage (like MinIO):
+
+```bash
+docker run --rm \
+  -p 8080:8080 \
+  -v $(pwd)/data:/var/moderne \
+  -e PUBLISH_URL=s3://your-bucket \
+  -e S3_PROFILE=default \
+  mass-ingest:quickstart
+```
+
+
+S3 environment variables:
+- `PUBLISH_URL` - S3 bucket URL (must start with `s3://`)
+- `S3_PROFILE` - AWS profile name for credentials (optional, uses AWS credential chain by default)
+- `S3_REGION` - AWS region for cross-region bucket access (optional)
+- `S3_ENDPOINT` - S3 endpoint URL (optional, for S3-compatible services like MinIO)
+
+**Note on AWS Credentials:**
+When running on AWS (EC2/ECS/Fargate), the container automatically uses IAM roles:
+- EC2: Uses instance profile
+- ECS/Fargate: Uses task role
+- No S3_PROFILE needed when IAM roles are configured
+
+For local testing in containers, use one of:
+
+**Option 1: Environment Variables (Simplest)**
+```bash
+docker run --rm \
+  -p 8080:8080 \
+  -v $(pwd)/data:/var/moderne \
+  -e PUBLISH_URL=s3://your-bucket/lsts/ \
+  -e AWS_ACCESS_KEY_ID=your-access-key \
+  -e AWS_SECRET_ACCESS_KEY=your-secret-key \
+  -e AWS_REGION=us-east-1 \
+  mass-ingest:quickstart
+```
+
+**Option 2: Mount AWS Credentials File**
+```bash
+docker run --rm \
+  -p 8080:8080 \
+  -v $(pwd)/data:/var/moderne \
+  -v ~/.aws:/root/.aws:ro \
+  -e PUBLISH_URL=s3://your-bucket/lsts/ \
+  -e S3_PROFILE=your-profile \
+  mass-ingest:quickstart
+```
+
+**Option 3: Use AWS SSO/CLI Credentials** (if using AWS SSO)
```

Comment:
This is _a way_ to do this, but the more typical way would be `aws sso login --profile your-profile`, then utilizing Option 2 directly (including the preference toward `AWS_PROFILE`).